iis-wf/iis-wf-export-actionmanager/src/main/java/eu/dnetlib/iis/wf/export/actionmanager/entity/AbstractEntityExporterProcess.java

```diff
@@ -43,225 +41,219 @@
 import eu.dnetlib.iis.common.java.porttype.PortType;
 import eu.dnetlib.iis.wf.export.actionmanager.api.ActionManagerServiceFacade;
 import eu.dnetlib.iis.wf.export.actionmanager.api.SequenceFileActionManagerServiceFacade;
-import eu.dnetlib.iis.wf.export.actionmanager.cfg.ActionManagerConfigurationProvider;
 import eu.dnetlib.iis.wf.export.actionmanager.cfg.StaticConfigurationProvider;
+import eu.dnetlib.iis.wf.export.actionmanager.entity.facade.MDStoreFacade;
+import eu.dnetlib.iis.wf.export.actionmanager.entity.facade.MDStoreFacadeFactory;
 
 /**
  * Abstract entity exporter process.
+ * 
```

Comment:
The same here, just the name of the class. No explanation what the service is for.